plugins/woocommerce-admin/client/products/fields/variations/variations.tsx

```diff
@@ -39,6 +40,8 @@ const NOT_VISIBLE_TEXT = __( 'Not visible to customers', 'woocommerce' );
 const VISIBLE_TEXT = __( 'Visible to customers', 'woocommerce' );
 const UPDATING_TEXT = __( 'Updating product variation', 'woocommerce' );
 
+const TRUNCATE_LENGTH = 32;
```

Comment:
Optional, but we could use the variation title limit constant exposed [here](https://github.com/woocommerce/woocommerce/blob/bfa2d4f3a790686c18b99c4cf0f6f6b9e15d6a28/plugins/woocommerce-admin/client/products/utils/get-product-variation-title.ts#L6).

Might need to rebase to grab those changes.

Reply:
done -> [4003867](https://github.com/woocommerce/woocommerce/pull/36134/commits/4003867ebc7b44073c584ef3187d86ed00760996)